### DISCUSSION TITLE
RTX-Remix runtime support

### DISCUSSION THREAD

**Discussion by razvanab** [WolfireGames/overgrowth#157] [Q&A]
I know I am asking for a lot here, but I would like to see at least some basic support for the RTX-Remix runtime.

https://github.com/NVIDIAGameWorks/rtx-remix

**Comment by kavika13** [comment]
Does RTX-Remix only work on nVidia cards, and is the SDK closed source/proprietary? Is it Windows-only?
If any of the above is true, I suspect this should not be implemented.

**Reply by razvanab** [reply]
I think it can be made externally without modifying the game code.
It works on AMD GPUs too; from what I read, the license appears to be the MIT License, and I think it is Windows only.

From what I understand, the RTX-remix in that link is a wrapper around DX9 games.

`The NVIDIA RTX Remix is a tool designed to remaster classic DirectX 8 and 9 games. It acts as a wrapper around these games, allowing for the replacement of the DirectX 9 DLLs. This enables the injection of remastered assets, enhanced lighting, and other modern graphical improvements into older games.`
